app/components/common/UniversalButton.tsx

```diff
@@ -8,21 +8,25 @@ export interface UniversalButtonProps extends ButtonHTMLAttributes<HTMLButtonEle
   type: 'submit' | 'button' | 'reset';
 }
 /**
- * - `onClick`: A function to call when the button is clicked.
- * - `label`: The text to display on the button.
- * This allows the component to be reused for different actions and labels.
+ *  - `id`: Unique identifier for the button.
+ *  - `label`: Text to display on the button.
+ *  - `classNames`: Object specifying CSS classes for styling.
```

Comment:
I was pretty confused between the mantine's classNames prop and the commonly used classNames hook

Reply:
Tks for the reminder! Just looked up mantine docs. It is pretty different from the regular classNames. Just updated the jsdoc.